tajo-catalog/tajo-catalog-common/src/main/proto/CatalogProtos.proto

```diff
@@ -407,3 +407,8 @@ message IndexResponse {
   required ReturnState state = 1;
   optional IndexDescProto indexDesc = 2;
 }
+
+message GetPartitionsResponse {
```

Comment:
Every message for rpc communication with catalog should locate in CatalogProtocol.proto.